clients/youtube_producer/js/yt_auth/yt_authDirective.js

```diff
@@ -4,23 +4,10 @@ module.exports = function ($window, $http, $interval, $log) {
 		restrict: 'E',
 		link: function(scope, element, attr){
```

Comment:
var openidIframe = document.createElement('iframe'); -> you do not append this element to the DOM, or have I not seen something? But somehow it is in the DOM, when you create it. So maybe you are right. For me the functionality is not visible for the user and the iFrame is not really rendered and that´s why it is application logic ;)